src/main/java/org/fenixedu/learning/servlets/FenixEduLearningContextListener.java

```diff
@@ -88,8 +87,11 @@ public void contextInitialized(ServletContextEvent sce) {
         });
         Signal.register(ExecutionCourse.ACRONYM_CHANGED_SIGNAL, (DomainObjectEvent<ExecutionCourse> event) -> {
             ExecutionCourseListener.updateSiteSlug(event.getInstance());
-
         });
+
+        Signal.register(ProfessorshipPermissions.PROFESSORSHIP_PERMISSIONS_CHANGED, (DomainObjectEvent<ProfessorshipPermissions> event) ->
```

Comment:
This requires an upgrade of the fenixedu-academic module due to changes in the API yet to be released.